addons/base_setup/res_config_view.xml

```diff
@@ -82,6 +82,11 @@
                         </div>
                     </group>
                     <group>
+                        <label for="id" string="Google Analytics"/>
```

Comment:
Do not define a void group here; use an xpath in website_version/views/res_config.xml.